lib/version_utils.pm

```diff
@@ -564,6 +564,7 @@ sub is_server {
     return 1 if is_sles4sap();
     return 1 if is_sles4migration();
     return 1 if get_var('FLAVOR', '') =~ /^Server/;
+    return 1 if get_var('FLAVOR', '') =~ /^DMS-QEMU/;
```

Comment:
why do we need this ?

Reply:
ok nevermind I just saw that `is_public_cloud` also returns 1 here and I have no idea why :see_no_evil:  and I have no idea why . please ignore :smile: